packages/invoices/src/OutboundShipment/DetailView/Footer/StatusChangeButton.tsx

```diff
@@ -0,0 +1,122 @@
+import React, { useState, useEffect } from 'react';
```

Comment:
don't particularly like how this turned out.. other ways I tried with arrays of statuses and indexes to try and make it more dry just seemed to be more confusing than helpful

Reply:
I can feel the temptation to make a tricksy array based method - and expect that it would be much harder to understand. what you've got is clear and easy to follow; it's not too big so I think it's great like it is